clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/wgscram/TransferWgsCramV1Metadata.scala

```diff
@@ -60,13 +60,17 @@ case class TransferWgsCramV1Metadata(
   readgroupLevelMetricsFiles: Option[List[URI]] = None
 ) extends TransferMetadata[TransferWgsCramV1Metadata] {
 
-  // Delete the cramPath.md5 file (foo.cram.md5 where foo.cram is cramPath).
+  // Delete the cramPath.md5 file only if a workspaceName is defined otherwise there will be no md5
+  // (foo.cram.md5 where foo.cram is cramPath).
   override def pathsToDelete: Seq[URI] =
     Seq.concat(
       cramPath,
       craiPath,
-      cramPath.map(
-        cp => URI.create(s"$cp${WgsCramExtensions.Md5ExtensionAddition}")
+      workspaceName.flatMap(
+        _ =>
```

Comment:
oh scalafmt

Reply:
I don't understand why this is here. (Oh, never mind. I get it.)